src/backend/ai-route-handlers.ts

```diff
@@ -0,0 +1,146 @@
+import { Request, Response } from 'express'
+import * as t from 'io-ts'
+
+import config from '../utils/config'
+import OpenAI, { APIError } from 'openai'
+import { logger } from '../utils/logger'
+import * as jsonSchema from './ai/content-type.json'
+import { mergeTextPluginsRecur } from './ai/merge-text-plugins-recur'
+import {
+  generateBeforeAfterPrompt,
+  generateSystemPrompt,
+} from './ai/generate-prompts'
+import { changePrompt, changeSystemPrompt } from './ai/change-prompts'
+
+const GenerateQuery = t.type({
+  prompt: t.string,
+  before: t.string,
+  after: t.string,
+})
+
+const openai = new OpenAI({ apiKey: process.env.OPENAI_API_KEY })
+
+export async function generateContent(req: Request, res: Response) {
+  if (config.ENVIRONMENT !== 'staging') {
```

Comment:
```suggestion
  if (config.ENVIRONMENT === 'production') {
```
In order to test it already in the dev environment.
Repeat in the other file.